@types/phaser3-rex-plugins/plugins/board/grid/quad/Quad.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module 'phaser3-rex-plugins/plugins/board/grid/quad/Quad' {
+    export default class QuadGrid extends Quad {
+        constructor(config: any);
+        sides: number;
+        get allDirections(): number[];
+        get halfDirections(): number[];
+        setType(type: any): QuadGrid;
+        setDirectionMode(mode: any): QuadGrid;
+        setOriginPosition(x: any, y: any): QuadGrid;
+        setCellSize(width: any, height: any): QuadGrid;
+
+        getGridPoints(tileX: any, tileY: any, points: any): any;
```

Comment:
There is also some other methods https://github.com/rexrainbow/phaser3-rex-notes/blob/master/plugins/board/grid/quad/Quad.js#L51-L64